terraform/modules/route53-logging/variables.tf

```diff
@@ -0,0 +1,11 @@
+tags = {
```

Comment:
Tags don't need to be supplied in a module, and won't work in this format. Consider having a variable that tags a map here instead.

Reply:
Put in variables